src/app/client/src/app/modules/public/module/group/components/explore-group/explore-group.component.ts

```diff
@@ -7,9 +7,14 @@ import { Component, OnInit } from '@angular/core';
 })
 export class ExploreGroupComponent implements OnInit {
   hideLoginContainer;
+  showWelcomePopup = true;
```

Comment:
Remove unnecessary life cycle methods and code in this component

Reply:
Resolved